### PR TITLE
Feature/isort

### PR DESCRIPTION
**Pull request by tjdcs**
# Summary

Code format enforcement is good, and we have a lot if import order issues with PEP8. 

Luckily I think isort can fix that.

### CODE REVIEW

**Comment by tjdcs** [comment]
Running CI on this branch to see how it does. My local environment thinks everything passes but has a few other errors related to my early adoption of MacOS Sonoma 14.0

File: pyproject.toml [hunk 1]
```diff
@@ -106,7 +117,7 @@ reportMissingImports = false
 reportMissingModuleSource = false
 reportUnboundVariable = false
 reportUnnecessaryCast = true
-reportUnnecessaryTypeIgnoreComment = true
+reportUnnecessaryTypeIgneComment = true
```
Comment (KelSolaar) [inline pyproject.toml:120]: Swallowed some characters here! :)

**Comment by tjdcs** [comment]
note this is a branch from #1209 which should be resolved first!z

**Comment by KelSolaar** [comment]
Great and thanks! I will be able to tick that one off from the bottomless stack of things todo:

image

**Comment by tjdcs** [comment]
> Great and thanks! I will be able to tick that one off from the bottomless stack of things todo:
> 

Woot!

I don't intend to make any changes so if you think the project toml and pre-commit config looks good then this is ready for  `develop`

**Pull request merged**